differences_between_some_packages_of_visualization.Rmd

```diff
@@ -0,0 +1,198 @@
+# Differences between some packages of visualizaiton
+
+Yudong Zhou and Yanbo Wang
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE, cache = TRUE)
+```
+
+```{r}
+library(ggplot2)
+library(plotly)
+library(tidyverse)
+library(dygraphs)
+library(leaflet)
+library(ggfortify)
+library(corrplot)
+# require(devtools)
+# install_github('recharts', 'taiyun')
+library(recharts) # must be installed from source
+```
+
+## 0 -- Surface
```

Comment:
please confirm that the index/number here is needed since the hashtags(##) will be automatically converted to indices (something like 1.1, 1.2). I would suggest remove all the numbers after the hashtags

Reply:
OK, so how can i change them, just open the project and change these indexes and then commit and push?

Reply:
Yes, that's correct. Simply change your own branch, commit it and push.

Reply:
Hi, i have deleted all the indexes and pushed, please help me check that.

Reply:
Alright, I will merge your PR now.